src/ast.c

```diff
@@ -213,549 +213,549 @@ WasmModuleField* wasm_append_module_field(struct WasmAllocator* allocator,
 
-    case WASM_EXPR_TYPE_BR_IF:
+    case WABT_EXPR_TYPE_BR_IF:
       CALLBACK(on_br_if_expr);
       break;
 
-    case WASM_EXPR_TYPE_BR_TABLE:
+    case WABT_EXPR_TYPE_BR_TABLE:
       CALLBACK(on_br_table_expr);
       break;
 
-    case WASM_EXPR_TYPE_CALL:
+    case WABT_EXPR_TYPE_CALL:
       CALLBACK(on_call_expr);
       break;
 
-    case WASM_EXPR_TYPE_CALL_INDIRECT:
+    case WABT_EXPR_TYPE_CALL_INDIRECT:
       CALLBACK(on_call_indirect_expr);
       break;
 
-    case WASM_EXPR_TYPE_COMPARE:
+    case WABT_EXPR_TYPE_COMPARE:
       CALLBACK(on_compare_expr);
       break;
 
-    case WASM_EXPR_TYPE_CONST:
+    case WABT_EXPR_TYPE_CONST:
       CALLBACK(on_const_expr);
       break;
 
-    case WASM_EXPR_TYPE_CONVERT:
+    case WABT_EXPR_TYPE_CONVERT:
       CALLBACK(on_convert_expr);
       break;
 
-    case WASM_EXPR_TYPE_CURRENT_MEMORY:
+    case WABT_EXPR_TYPE_CURRENT_MEMORY:
       CALLBACK(on_current_memory_expr);
       break;
 
-    case WASM_EXPR_TYPE_DROP:
+    case WABT_EXPR_TYPE_DROP:
       CALLBACK(on_drop_expr);
       break;
 
-    case WASM_EXPR_TYPE_GET_GLOBAL:
+    case WABT_EXPR_TYPE_GET_GLOBAL:
       CALLBACK(on_get_global_expr);
       break;
 
-    case WASM_EXPR_TYPE_GET_LOCAL:
+    case WABT_EXPR_TYPE_GET_LOCAL:
       CALLBACK(on_get_local_expr);
       break;
 
-    case WASM_EXPR_TYPE_GROW_MEMORY:
+    case WABT_EXPR_TYPE_GROW_MEMORY:
       CALLBACK(on_grow_memory_expr);
       break;
 
-    case WASM_EXPR_TYPE_IF:
+    case WABT_EXPR_TYPE_IF:
       CALLBACK(begin_if_expr);
-      CHECK_RESULT(wasm_visit_expr_list(expr->if_.true_.first, visitor));
+      CHECK_RESULT(wabt_visit_expr_list(expr->if_.true_.first, visitor));
       CALLBACK(after_if_true_expr);
-      CHECK_RESULT(wasm_visit_expr_list(expr->if_.false_, visitor));
+      CHECK_RESULT(wabt_visit_expr_list(expr->if_.false_, visitor));
       CALLBACK(end_if_expr);
       break;
 
-    case WASM_EXPR_TYPE_LOAD:
+    case WABT_EXPR_TYPE_LOAD:
       CALLBACK(on_load_expr);
       break;
 
-    case WASM_EXPR_TYPE_LOOP:
+    case WABT_EXPR_TYPE_LOOP:
       CALLBACK(begin_loop_expr);
-      CHECK_RESULT(wasm_visit_expr_list(expr->loop.first, visitor));
+      CHECK_RESULT(wabt_visit_expr_list(expr->loop.first, visitor));
       CALLBACK(end_loop_expr);
       break;
 
-    case WASM_EXPR_TYPE_NOP:
+    case WABT_EXPR_TYPE_NOP:
       CALLBACK(on_nop_expr);
       break;
 
-    case WASM_EXPR_TYPE_RETURN:
+    case WABT_EXPR_TYPE_RETURN:
       CALLBACK(on_return_expr);
       break;
 
-    case WASM_EXPR_TYPE_SELECT:
+    case WABT_EXPR_TYPE_SELECT:
       CALLBACK(on_select_expr);
       break;
 
-    case WASM_EXPR_TYPE_SET_GLOBAL:
+    case WABT_EXPR_TYPE_SET_GLOBAL:
       CALLBACK(on_set_global_expr);
       break;
 
-    case WASM_EXPR_TYPE_SET_LOCAL:
+    case WABT_EXPR_TYPE_SET_LOCAL:
       CALLBACK(on_set_local_expr);
       break;
 
-    case WASM_EXPR_TYPE_STORE:
+    case WABT_EXPR_TYPE_STORE:
       CALLBACK(on_store_expr);
```

Comment:
Why not WABT_CALLBACK to avoid using what may be a common macro name. If you are doing it for constants, why not the (non-constant) macro?